string t_now
float32 x
float32 y
float32 theta
float32 transitional_velocity
float32 rotational_velocity
float32 t_delta
